### ISSUE TITLE
Workflow automation: editor

### ISSUE THREAD

**Issue by biapar** [ovh/public-cloud-roadmap#409]
Is there an editor to manage the workflows?

**Comment by DavidDelebecque** [comment]
Hi, 

Not sur about which workflow you're talking about ?
In the manager, we have mainly a workflow to schedule and automate backup instances.

**Comment by biapar** [comment]
Hi, Is't not possible to schedule other type of wf?

**Comment by DavidDelebecque** [comment]
no, if you want to automate, you can use API or Infra as Code tooling
